configurations/morcore2esuites/Configuration_TaakApplicatieApi.xml

```diff
@@ -5,42 +5,65 @@
             <ApiListener
                 name="ListTaakTypes"
                 uriPattern="/taaktypes" />
-
         </Receiver>
 
         <Pipeline>
             <Exits>
                 <Exit name="Exit" state="SUCCESS" />
-            </Exits>
-
-            <EchoPipe name="Hello" getInputFromFixedValue="World!"></EchoPipe>
+                <Exit name="Reject" state="REJECTED"/>
+                <Exit name="failure" state="ERROR"/>
+             </Exits>
+            
+            <fixedResultPipe name="InputTaaktypesLijst" filename="responses/ZaakTypes.xml"/>
+
+            <XmlValidatorPipe name="ValidateInputMessage" schema="xsd/validateGetTaakTypes.xsd" root="root"/>
+                <Forward name="jsonpipe" path="XmlToJson"/>
+                <Forward name="failure" path="failure"/>
+            
+            <JsonPipe name="XmlToJson" direction="XML2JSON">
+                <Forward name="success" path="Exit"/>
+                <Forward name="failure" path="Reject"/>
+            </JsonPipe>
         </Pipeline>
     </Adapter>
 
     <Adapter name="Action_GetTaakType">
         <Receiver name="GetTaakType">
             <ApiListener
                 name="GetTaakType"
-                uriPattern="/taaktypes/{uuid}" />
-
+                uriPattern="/taaktypes/{uuid}"/>
         </Receiver>
 
         <Pipeline>
             <Exits>
                 <Exit name="Exit" state="SUCCESS" />
-            </Exits>
+                <Exit name="Reject" state="REJECTED"/>
+                <Exit name="failure" state="ERROR"/>
```

Comment:
For consistency it is better to set the failure Exit name to "Exception" (mind the capitalization).
Every pipe has a forward called failure which is an indication of an error occurring within the pipe or a condition not being met. By renaming it to Exception you make it clear that what you're about to do is Error-related and not related to handling a conditional failure (such as an if statement not passing).